.htmltest.yaml

```diff
@@ -4,3 +4,4 @@
 IgnoreDirectoryMissingTrailingSlash: true
 DirectoryPath: spec
 CheckExternal: false
+IgnoreInternalEmptyHash: true
```

Comment:
for the record: this stops `htmltest` complaining about links with `href="#"`

Reply:
htmltest only tests the HTML source, the ToC was generated with JavaScript, so htmltest didn't check it.

Reply:
oh of course